llvm/include/llvm/CodeGen/BasicTTIImpl.h

```diff
@@ -1468,7 +1466,7 @@ class BasicTTIImplBase : public TargetTransformInfoImplCRTPBase<T> {
   }
 
   InstructionCost getMemoryOpCost(
-      unsigned Opcode, Type *Src, MaybeAlign Alignment, unsigned AddressSpace,
+      unsigned Opcode, Type *Src, Align Alignment, unsigned AddressSpace,
```

Comment:
This is probably the only change worth paying attention to.
Commit that changed TTI interface (but not implementations): 27b4e6931d0f2f16293e6abface7da25fb731e14

Reply:
MaybeAlign shouldn't exist, Align(1) is always a suitable value